File: src/test/java/no/nav/fo/veilarbregistrering/besvarelse/StillingTestdataBuilder.java
package no.nav.fo.veilarbregistrering.besvarelse;

public class StillingTestdataBuilder {
    public static Stilling gyldigStilling() {
        return new Stilling()
                .setStyrk08("12345")
                .setLabel("yrkesbeskrivelse")
                .setKonseptId(1246345L);
    }

    public static Stilling ingenYrkesbakgrunn() {
        return new Stilling("X", -1L, "X");
    }
}

File: src/test/java/no/nav/fo/veilarbregistrering/besvarelse/StillingTestdataBuilder.kt
package no.nav.fo.veilarbregistrering.besvarelse

object StillingTestdataBuilder {
    @JvmStatic
    fun gyldigStilling(
        styrk08: String = "12345",
        label: String? = "yrkesbeskrivelse",
        konseptId: Long = 1246345L,
    ): Stilling =
        Stilling(styrk08 = styrk08, label = label, konseptId = konseptId)

    @JvmStatic
    fun ingenYrkesbakgrunn(): Stilling {
        return Stilling("X", -1L, "X")
    }
}
